src/packagers/JSConcatPackager.js

```diff
@@ -154,36 +168,36 @@ class JSConcatPackager extends Packager {
 
     await this.write('});');
 
-    super.write(
-      this.buffer
-        .replace(
-          /\$parcel\$expand_exports\(([\d]+),([\d]+)\)/g,
-          (_, from, to) => {
-            const exports = this.exports.get(Number(from));
-
-            if (!exports) {
-              throw new Error();
-            }
-
-            return exports
-              .map(
-                name =>
-                  `var $${to}$named_export$${name} = $${from}$named_export$${name}`
-              )
-              .join('\n');
-          }
-        )
-        .replace(/\$([\d]+)\$named_export\$([^ ]+) =/g, (binding, id, name) => {
-          const exportBinding = `$${id}$exports`;
-
-          // if there is at least two occurences of the exports binding
-          if (this.buffer.split(exportBinding).length > 2) {
-            return `${binding} ${exportBinding}.${name} =`;
-          }
-
-          return binding;
-        })
-    );
+    // super.write(
```

Comment:
I think these re-exports are going to be problematic since they are supposed to be live bindings rather than a copy. Commented for now while we figure it out.